tcms/profiles/tests.py

```diff
@@ -30,7 +30,7 @@ def test_user_can_view_their_own_profile(self):
         self.assertContains(response, self.tester.username)
         self.assertContains(response, self.tester.email)
 
-    def test_user_case_view_profile_of_another_user(self):
+    def test_view_peregrine_profile(self):
```

Comment:
What does `peregrine` means? Haven't heard the word, and found nothing on the internet in that context. Maybe `test_view_another_user_profile` or `test_view_another_profile`

Reply:
On a side note:  having meaningful test names is fine but don't try too hard to make them fit in 30 characters to please pylint. For test files you can fix all the invalid names inside the code like variables, etc and then just ingore the invalid-name checker. Either per method if there are just a few of them or for the entire module if there are too many methods that need renaming.

Now that said in this case you should remove this comit from the PR and ammend the previous commit to disable the invalid-name checker for this file (I assume everything else is OK with this file).